packages/client/lib/blockchain/chain.ts

```diff
@@ -414,6 +471,22 @@ export class Chain {
     return this.blockchain.getCanonicalHeadBlock()
   }
 
+  /**
+   * Gets the latest block in the canonical chain
+   */
+  async getCanonicalSafeBlock(): Promise<Block> {
```

Comment:
Why do we have separate functions if they do the same thing?  Are we going to get to a place in our code where we might have different safe or finalized blocks?

Reply:
safe and finalized are not same... safe can become finalized (or in rare re-org scenarios not)

Reply:
Ah, sorry, misread the code.